### DISCUSSION TITLE
Longhorn Engine shoing incompatible - Isues Upgrading Longhorn Volumes After EKS Upgrade

### DISCUSSION THREAD

**Discussion by othapall** [longhorn/longhorn#9634] [Q&A] [answered]
Hi everyone,

I've recently upgraded my AWS EKS cluster from version 1.29 to 1.30 and also upgraded Longhorn from version 1.5.3 to 1.6.2. However, I'm encountering an issue where I'm unable to upgrade the Longhorn volumes to the new engine version (1.6.2). The engine status shows as "incompatible."

Details:
EKS Version: 1.30
Longhorn Version: 1.6.2
Previous Longhorn Version: 1.5.3

Steps Taken:
Upgraded EKS cluster to 1.30.
Upgraded Longhorn to 1.6.2 via the Longhorn UI.
Attempted to upgrade existing Longhorn volumes to use the new engine.

Observations:
The Longhorn UI indicates that the engine version is incompatible with the current volumes.
UI Showing error failed to get the parameters: failed to get target node ID: failed to find a node that is ready and has the default engine image dockerhub/longhornio/longhorn-engine:v1.6.2 deployed.

All the pods running fine on longhorn namesapce.

Questions:
Is there a known issue with upgrading Longhorn volumes from 1.5.3 to 1.6.2 after an EKS upgrade?
Are there any recommended steps to resolve the incompatibility and successfully upgrade the volumes?
Should I consider any specific settings or configurations during the upgrade process?
Any insights or guidance would be greatly appreciated!

Thanks in advance!

**Comment by c3y1huang** [comment, marked as answer]
Discussed in https://github.com/longhorn/longhorn/issues/9635.
